src/app/shared/helpers/crew.mapper.ts

```diff
@@ -0,0 +1,16 @@
+import { MediaType } from '../enums/media-types';
+import {
+  CastCard,
+  CastDTO
+} from '../interfaces/cast';
+
+export function castCrewMapper(obj: CastDTO): CastCard {
+  return {
+    id: obj.id ?? 0,
+    name: obj.title || '',
```

Comment:
nit: Why you sometimes use `??` and sometimes '||'? It's better to be consistent I believe